server/routes/accounts/user.js

```diff
@@ -474,6 +474,10 @@ const partAddUser = async (req, res) => {
   }
 };
 
+const notPermittedToRegisterNewUser = (role) => !role || role === 'Read' || role === 'None';
+
+const newUserRoleNotValid = (role) => !role || !['Edit', 'Read', 'None'].includes(role);
```

Comment:
It would never get to this as it would be hit on the if above?
```suggestion
const newUserRoleNotValid = (role) => !['Edit', 'Read', 'None'].includes(role);
```